docs/framework/wcf/feature-details/uritemplate-and-uritemplatetable.md

```diff
@@ -36,9 +36,9 @@ Web developers require the ability to describe the shape and layout of the URIs
   
  <xref:System.UriTemplate> is designed to work with any URI scheme that conforms to the HTTP URI grammar. The following are examples of supported URI schemes.  
   
-- http://  
+- `http://`  
   
-- https://  
+- `https://`  
```

Comment:
I think that the other items in this list need to be formatted the same or these two need to be escaped some how so as to not cause any formatting.

Reply:
Added formatting to the others